data_dir/scylla-dash-per-server-nemesis.master.json

```diff
@@ -6792,7 +6792,7 @@
       ]
     },
     "timezone": "utc",
-    "title": "[longevity-twcs-3h] Scylla Per Server Metrics Nemesis Master",
+    "title": "[$test_name] Scylla Per Server Metrics Nemesis Master",
     "uid": "J1T2FZ57za",
```

Comment:
IIUC this should be removed

Reply:
You mean uid?

Reply:
yes, it was added in the same commit where the "title" changed, so i think it should be removed

Reply:
Let's remove it and run provision test on it, to see if it will be able to restore monitoring stack

Reply:
done

> "longevity-twcs-3h"

done